### PR TITLE
Zenarmor Documentation Changes

### PR DESCRIPTION
**Pull request by zenarmor**
Hello,

I edited Sunny Valley Page .
Updated some of the links from Zenarmor docs.
Murat Balaban @muratbalaban43 is informed for these changes.

Regards.

### CODE REVIEW

File: source/vendor/sunnyvalley/zenarmor_hardwarerequirements.rst [hunk 1]
```diff
@@ -4,9 +4,8 @@ Zenarmor (Sensei): Hardware Requirements
 
 Due to the nature of deep packet analysis and detailed drill-down reporting functionality, Zenarmor requires more hardware resources than a standard L3-L4 firewall.
 
-
-**.. Note:: :**
-
+ **Note**
```
Comment (AdSchellevis) [inline source/vendor/sunnyvalley/zenarmor_hardwarerequirements.rst:7]: you probably want to change this to :

.. Note::
     
for an example, see https://raw.githubusercontent.com/opnsense/docs/master/source/manual/aliases.rst

File: source/vendor/sunnyvalley/zenarmor_hardwarerequirements.rst [hunk 2]
```diff
@@ -18,7 +17,7 @@ CPU & Memory
 
 Because the analytics module relies on Elasticsearch to process large amounts of data, the amount of the memory available in the system is crucial for the overall performance of Zenarmor.
 
-**.. Tip:: :**
+**Tip**
```
Comment (AdSchellevis) [inline source/vendor/sunnyvalley/zenarmor_hardwarerequirements.rst:20]: same

File: source/vendor/sunnyvalley/zenarmor_hardwarerequirements.rst [hunk 3]
```diff
@@ -33,8 +32,8 @@ Below is the recommended minimum hardware requirements for Zenarmor based on the
  250-1000              500 Mbps - 100 Kpps        32 GB               Intel Quad-Core i7 3.4 GHz (4 Cores, 8 Threads) or equivalent
 =====================  =========================  ==================  ======================================================================
 
-**.. Note:: :**
-
+ **Note**
```
Comment (AdSchellevis) [inline source/vendor/sunnyvalley/zenarmor_hardwarerequirements.rst:35]: same

File: source/vendor/sunnyvalley/zenarmor_install.rst [hunk 4]
```diff
@@ -2,9 +2,13 @@
 Zenarmor (Sensei): Installing via Web Interface
 ==================================================
 
-.. Note::
 
-    Zenarmor Free Edition is **forever free-of-charge**. We strongly recommend you register to keep in touch with updates and new features. You can register at `https://www.sunnyvalley.io/sensei <https://www.sunnyvalley.io/sensei>`_
+
+
+ **Note**
```
Comment (AdSchellevis) [inline source/vendor/sunnyvalley/zenarmor_install.rst:8]: same

**Comment by AdSchellevis** [comment]
@zenarmor did you compile these locally? it might help to have sphinx docs installed locally to find compile issues easily

File: source/vendor/sunnyvalley/zenarmor_hardwarerequirements.rst (hunk 1 above)
Comment (zenarmor) [reply source/vendor/sunnyvalley/zenarmor_hardwarerequirements.rst:7]: Actually iI wanted to add note icon like   https://github.com/community/community/discussions/16925

Comment (AdSchellevis) [reply source/vendor/sunnyvalley/zenarmor_hardwarerequirements.rst:7]: @zenarmor ok, no problem, if this is what you'er looking for, I'll merge

**Pull request merged**